pkg/binding/controller.go

```diff
@@ -85,7 +84,7 @@ type Controller struct {
 
 	extClient             apiextensionsclientset.Interface // used for CRD
 	listers               map[string]cache.GenericLister
-	gvkGvrMapper          util.GvkGvrMapper
+	gvkToGvrMapper        util.GVKToGVRMapper
 	informers             map[string]cache.SharedIndexInformer
 	stoppers              map[string]chan struct{}
```

Comment:
FYI: You can now replace these strings with GroupVersionKind or GroupVersionResource. This would make the code more reliably checkable at compile time.